API/Hardware_API.md

```diff
@@ -68,6 +68,61 @@ If you're newer to hardware and these functions look like alphabet soup to you,
 
 By default, all of the pins are pulled high if not specifically set.
 
+### Pullup and Pulldown pins
+
+Pins 2-7 on both Ports are available for pullup and pulldown.
+
+The basic function of a pull-up resistor is to insure that given no other input, a circuit assumes a default value. The 'pullup' mode pulls the line high and the 'pulldown' mode pulls it low.
+
+### Usage
+```js
+// Invoke this method to set the pull mode. pullType can be - pullup, pulldown or none.
+pin.pull(pullType, callback);
+```
+
+Example of the `pin.pull(mode)` command using a pushbutton. The code example given below turns on the Blue LED of the Tessel module when the pushbutton is pressed and turns off the Blue LED when the pushbutton is released.
+
+```js
+var tessel= require('tessel'); // Import Tessel
+
+var pin = tessel.port.A.pin[2]; // Select pin 2 on port A
+
+var pullType = "pullup"; // Set the mode of `pin.pull to pullup`
```

Comment:
Can you change this to single quotes, `'pullup'` instead of `"pullup"`, to keep it consistent in the sample code?